### PR TITLE
travis: run tests on in-memory cozy-stack

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -71,6 +71,7 @@ before_install:
 
     # Cozy stack v3
     - travis_retry go get -u github.com/cozy/cozy-stack
+    - cp ./dev/cozy-stack/cozy.yaml $HOME/cozy.yaml
     - $GOPATH/bin/cozy-stack serve --log-level warning >/dev/null &
```
Comment (nono) [inline .travis.yml]: Maybe you can use `--fs-url` here or move the log level option to the config file. It looks weird to have configuration from both the command-line flags and the config file.

File: test/support/helpers/cozy.js [hunk 2]
```diff
@@ -53,7 +53,8 @@ async function rootDirContents () {
 async function deleteAll () {
   const docs = await rootDirContents()
 
-  await Promise.all(docs.map(doc => cozy.files.trashById(doc._id)))
+  await Promise.all(docs.map(doc => cozy.files.trashById(doc._id)
+      .catch(_ => true))) // ignore all errors
```
Comment (sebn) [inline test/support/helpers/cozy.js]: Out of curiosity, which errors are we getting otherwise?

**Comment by aenario** [comment]
Restarting build following : https://github.com/cozy/cozy-stack/pull/1545 Thanks @nono

**Comment by aenario** [comment]
Reported.

**Pull request closed without merging**